builder/vmware/common/driver.go

```diff
@@ -132,6 +163,15 @@ var dhcpConfPaths = []string{
 	"dhcpd/dhcpd.conf",
 }
 
+// The file extensions to retain when cleaning up files in a virtual machine environment.
+var fileExtensions = []string{
```

Comment:
Super duper tiny nit, but I would rename this variable, KeepFileExtensions is a bit clumsy, but it's still more communicative than fileExtensions on what this constant is used for.

I tend to write overly verbose variables in situations like this, which people may or may not disagree with, but I'd make this something like skipCleanFileExtensions or fileExtensionsSkipClean